alphacsc/convolutional_dictionary_learning.py

```diff
@@ -341,20 +341,23 @@ class BatchCDL(ConvolutionalDictionaryLearning):
     __doc__ = DOC_FMT.format(**_default)
 
     def __init__(self, n_atoms, n_times_atom, reg=0.1, n_iter=60, n_jobs=1,
+                 loss='l2', loss_params=dict(gamma=.1, sakoe_chiba_band=10,
+                                             ordar=10),
```

Comment:
I would actually remove these parameters as the code for `loss != 'l2'` is not really documented.

Reply:
As we check the parameters when the solver is `dicodlle` I've put this in order to be able to set loss_params=None. Otherwise we should set loss_params=None everywhere.

Reply:
Ok I get it! I would say that having to set `loss_params=None` in every call seems sub-optimal as there is a high probability it is never used.

So if we could have a default to `None` and set the default value to `dict(gamma=.1, sakoe_chiba_band=10, ordar=10)` if `loss != 'l2'` at the beginning of `learn_d_z_multi`, I guess this would make it easier? WDYT?

Reply:
Ok!